svc/FetchService.js

```diff
@@ -104,29 +113,50 @@ export class FetchService {
     /**
      * Send an HTTP request to a URL, and decode the response as JSON.
      *
-     * @param {Object} opts - options to pass through to fetch, with some additions.
-     *      @see https://developer.mozilla.org/en-US/docs/Web/API/Request for the available options
-     * @param {string} opts.url - target url to send the HTTP request to. Relative urls will be
-     *     appended to XH.baseUrl for the request
-     * @param {Object} [opts.params] - parameters to encode and send with the request body (for POSTs)
-     *      or append as a query string.
-     * @param {string} [opts.method] - The HTTP Request method to use for the request. If not
-     *     explicitly set in opts then the method will be set to POST if there are params,
-     *     otherwise it will be set to GET.
-     * @param {string} [opts.contentType] - value to use in the Content-Type header in the request.
-     *     If not explicitly set in opts then the contentType will be set based on the method. POST
-     *     requests will use 'application/x-www-form-urlencoded', otherwise 'text/plain' will be
-     *     used.
+     * This method delegates to @see {fetch} and accepts the same options.
```

Comment:
Note that the `@see {foo}` annotation here doesn't really do anything - jsDoc needs those to be on their own line to be parsed / effective - and most likely to use the namespaced version `FetchService#fetch`

We will need to see what we end up doing w.r.t. docs like this.  It's unfortunate not to have the doc on the method that is most commonly used, as that is where you would get assistance and inline help from an editor. I use that consistently in my own development and get a lot of benefits from it. (I believe I had copied the relevant docs myself to provide this.)

We can revisit as we go. Just please be aware. Yes, this might in the end involve some careful duplication of comments, but I think we should treat the doc comments like we do any other code, reviewing them carefully and keeping them up to date.